edb/edgeql/compiler/schemactx.py

```diff
@@ -30,6 +30,8 @@
     Union,
     Iterable,
     Sequence,
+    List,
```

Comment:
We are on Python 3.11+ so prefer using builtin types instead, e.g `list[foo]` instead of `List[foo]`